### DISCUSSION TITLE
private key is visible

### DISCUSSION THREAD

**Discussion by 5hr1ganesh** [Cyfrin/foundry-full-course-cu#2946] [General]
### Course:Foundry Fundamentals, Section:2 Foundry Fund Me, Lesson:23
@PatrickAlphaC
the private key of the demo account being used is visible for a second in the video.

I understand that the account does not have any Mainnet ETH, but considering how difficult it has become to get Sepolia ETH nowadays(for free users), i just wanted to let it be known, if by a slight chance, it was missed.

**Comment by LordAdhaar** [comment]
https://cloud.google.com/application/web3/faucet/ethereum/sepolia

Use this

**Reply by 5hr1ganesh** [reply]
<img width="745" alt="image" src="https://github.com/user-attachments/assets/92541671-fbdc-4751-936b-6b892c798126">

i have used the service perhaps 1-2 times

**Reply by LordAdhaar** [reply]
I got some, how much you need

**Reply by 5hr1ganesh** [reply]
Nah I have barely used any of the 0.02 sepolia ETH I got from the faucet, earlier.

**Comment by EngrPips** [comment]
> ### Course:Foundry Fundamentals, Section:2 Foundry Fund Me, Lesson:23
> @PatrickAlphaC the private key of the demo account being used is visible for a second in the video.
> 
> I understand that the account does not have any Mainnet ETH, but considering how difficult it has become to get Sepolia ETH nowadays(for free users), i just wanted to let it be known, if by a slight chance, it was missed.

Thanks for the heads-up Friend.
